js/paymentPlanToggler.js

```diff
@@ -1,298 +1,304 @@
-CRM.$(function ($) {
-  /**
-   * Perform changes on form to add payment plan as an option to pay for
-   * membership.
-   */
-  $(function () {
-    moveMembershipFormFields();
-    setMembershipFormEvents();
-    initializeMembershipForm();
-  });
+function paymentPlanToggler(togglerValue, currencySymbol) {
 
-  /**
-   * Leaves form in a consistent state after the fields have been moved and the
-   * new events added.
-   */
-  function initializeMembershipForm () {
-    selectPaymentPlanTab(togglerValue);
-  }
+  CRM.$(function ($) {
```

Comment:
Should we remove this line ?

Reply:
We would like to keep it as it is to ensure that elements are loaded and they are in CRM. context.

Reply:
You use it inside `CRM.$(function ($) {` so you have the assurance , It should be fine.

Reply:
@ahed-compucorp  It needs a closure here otherwise, I would need to replace $ with CRM.$